_LowLevelCode/cpp/test/combine_sqw.tests/combine_sqw.test.cpp

```diff
@@ -497,41 +497,45 @@ TEST_F(TestCombineSQW, SQW_Reader_Propagate_Pix) {
   std::size_t pix_start_num, num_bin_pix, start_buf_pos(0);
   std::vector<float> pix_buffer(NUM_PIXBLOCK_COLS * 1000);
 
+  std::size_t bin_number{0};
   reader.get_pix_for_bin(0, pix_buffer.data(), start_buf_pos, pix_start_num,
```

Comment:
The zero here is `bin_number`, since you have that variable declared now.

Reply:
🤦‍♂